### ISSUE TITLE
Pls disable autocomplete on user input fields

### ISSUE THREAD

**Issue by arnehorn** [be-gosu/issues#252]
When you wanna invite user to a channel you type in the respective name. Usually you get a autocomplete suggestion and ignore it. Most of the times (I think when you use the mouse curser) when you choose a user via the autocomplete list I think the following happens:

Through the click on f.e. zGrav in the autocomplete list the field is updated with the zGrav User name and afterwards overridden by the autocomplete of the OS. This leads to the feeling of autocomplete is broken.

Long story short: disable autocomplete in fields where user names might be present
